flight/PiOS/STM32F30x/pios_usb_cdc.c

```diff
@@ -386,22 +385,10 @@ static void PIOS_USB_CDC_CTRL_EP_IN_Callback(void)
 	bool valid = PIOS_USB_CDC_validate(usb_cdc_dev);
 	PIOS_Assert(valid);
 
-	/* Give back UART State Bitmap */
-	/* UART State Bitmap
-	 *   15-7: reserved
-	 *      6:  bOverRun    overrun error
-	 *      5:  bParity     parity error
-	 *      4:  bFraming    framing error
-	 *      3:  bRingSignal RI
-	 *      2:  bBreak      break reception
-	 *      1:  bTxCarrier  DSR
-	 *      0:  bRxCarrier  DCD
-	 */
-	uart_state.bmUartState = htousbs(0x0003);
-
-	UserToPMABufferCopy((uint8_t *) &uart_state,
+	UserToPMABufferCopy((const uint8_t *) &uart_state,
```

Comment:
This changed nothing except removing documentation and putting the struct in flash/not writing it each time? :/